docs/serving/README.md

```diff
@@ -72,9 +72,8 @@ For more information on the resources and their interactions, see the [Resource
 
 ## Configuration and Networking
 
-- [Configuring cluster local routes](./cluster-local-route)
+- [Configuring cluster local routes](./deploying-from-private-registry.md)
```

Comment:
I think this one should be linking to https://deploy-preview-3728--dev-knative.netlify.app/development/developer/serving/services/private-services/ ?